components/service/glean/docs/pings/pings.md

```diff
@@ -46,28 +58,13 @@ The following fields are included in the `client_info` section, for every ping.
 All the metrics surviving application restarts (e.g. `client_id`, ...) are removed once the
 application using glean is uninstalled.
 
-### A note about time formats
-Time formats are used and expected in ISO 8601 format.  Due to the minimum Android SDK version 21
-not having direct support of outputting or parsing these formats, there was difficulty in finding
-a way to output these formats with the `:` character properly encoded in the timezone offset.  So
-we get the following:
-
-```
-2018-12-19T12:36-0600
-```
-
-We require the following to comply with certain back-end services:
-
-```
-2018-12-19T12:36-06:00
-```
-
-Which is why the `:` is manually inserted in order to comply with the format and requirements.
```

Comment:
I felt like this section wasn't really relevant in *user* docs.  It's just an implementation detail (which is also documented in comments in the relevant part of the code).  We also describe the date format in the datetime metric section now.

Reply:
Mh, you have a point. My concern is that "pipeline users" might need this reference in case things go wrong. But we can still look up the code, if needed.

Reply:
I see -- you just mean in case our code stops doing this correctly (i.e. the workaround stops working?)

Reply:
Yes, or in case something is changed on the pipeline and this requirement becomes no longer relevant